paddle.py

```diff
@@ -0,0 +1,20 @@
+from turtle import Turtle
+
+
+class Paddle(Turtle):
+
+    def __init__(self, position):
+        super().__init__()
+        self.shape("square")
+        self.color("white")
+        self.shapesize(stretch_wid=5, stretch_len=1, outline=0.5)
+        self.penup()
+        self.goto(position)
+
```

Comment:
It is better to add some comments for the methods

Reply:
comments added